libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Storage/KeyValueEncryptedFileStore.swift

```diff
@@ -59,6 +89,13 @@ public class KeyValueEncryptedFileStore: NSObject {
         self.name = name
         self.directory = URL(fileURLWithPath: fullPath)
         self.encryptionKey = encryptionKey
+        super.init()
+        
+        if isNewlyCreated {
+            _ = writeFile(key: KeyValueEncryptedFileStore.storeVersionFileName,
```

Comment:
Maybe writeFile (and readFile) should have a discardable result?

Reply:
`writeFile` makes sense. Thanks!

Reply:
ugh, I meant to say "(and removeFile)", that would make more sense :P

Reply:
Makes sense too! Updated.